activestorage/lib/active_storage/reflection.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module ActiveStorage
+  module Reflection
+    # Holds all the metadata about a has_one_attached attachment as it was
+    # specified in the Active Record class.
+    class HasOneAttachedReflection <
+        ::ActiveRecord::Reflection::MacroReflection #:nodoc:
```

Comment:
Why you need the `::` here? Also keep everything in the same line.

Reply:
Top level constant just for the milliseconds it would shave off the constant lookup - I'll remove